packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -225,51 +225,50 @@ class SlideTransition extends AnimatedWidget {
   }
 }
 
-/// Animates the scale of a transformed widget.
+/// Signature for the callback to [MatrixTransition.onTransform].
 ///
-/// Here's an illustration of the [ScaleTransition] widget, with it's [alignment]
-/// animated by a [CurvedAnimation] set to [Curves.fastOutSlowIn]:
-/// {@animation 300 378 https://flutter.github.io/assets-for-api-docs/assets/widgets/scale_transition.mp4}
-///
-/// {@tool dartpad}
-/// The following code implements the [ScaleTransition] as seen in the video
-/// above:
+/// Computes a [Matrix4] to be used in the [MatrixTransition] transformed widget
+/// from the [MatrixTransition.animation] value.
+typedef TransformCallback = Matrix4 Function(double);
```

Comment:
nit: please give the function argument a name (e.g.

```suggestion
typedef TransformCallback = Matrix4 Function(double animationValue);
```

(This makes sure autocomplete in IDEs works better by suggesting proper names for the callback arguments instead of something like arg0)